src/mpid/ch4/src/ch4_vci.c

```diff
@@ -0,0 +1,71 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ * (C) 2019 by Argonne National Laboratory.
+ *     See COPYRIGHT in top-level directory.
+ */
+
+#include "mpidimpl.h"
+#include "ch4_vci.h"
+
+#ifndef MPIDI_CH4_DIRECT_NETMOD
+int MPIDI_vci_pool_alloc(int num_vsis, int num_vnis)
+#else
```

Comment:
What is the purpose of `num_vcis`?  Is it to allocate these many vcis with vsi capability?